Tensile/Tests/pre_checkin/mfma/dgemm_64bit_offset.yaml

```diff
@@ -0,0 +1,64 @@
+TestParameters:
+  marks: [skip-gfx900, skip-gfx906, skip-gfx908, skip-gfx1010, skip-gfx1011, skip-gfx1012, skip-gfx1030, skip-gfx1100, skip-gfx1101, skip-gfx1102] # not supported by arch
+
+GlobalParameters:
+  NumElementsToValidate: -1
+  BoundsCheck: True
+  KernelTime: True
+  # PrintSolutionRejectionReason: True
+  BufferOffsetB: 4294967296
+  PristineOnGPU: False
+
+BenchmarkProblems:
+  ########################################
+  # NT - standard
+  ########################################
+  - # sgemm TN
+    - # ProblemType
+      OperationType: GEMM
+      DataType: d
+      TransposeA: False
+      TransposeB: False
+      UseBeta: True
+      Batched: True
+
+    - # BenchmarkProblemSizeGroup - Standard
+      InitialSolutionParameters:
+      BenchmarkCommonParameters:
+        - KernelLanguage: ["Assembly"]
+        - EdgeType: ["ShiftPtr"]
+        - PrefetchLocalRead: [True]
+      ForkParameters:
+        - AssertMinApproxSize: [0]
+        - DepthU: [8]
+        - ExpandPointerSwap: [0]
+        - GroupLoadStore: [1]
+        - LocalReadVectorWidth: [1]
+        - MACInstruction: ["MAD"]
```

Comment:
Could you please use FMA here to make thing easier in the future?

Reply:
Or, just remove this line.

Reply:
I used "MAD" setting here to match a specific test case from another project that saw an error. This setting shouldn't really affect the test, but just wanted to be sure the requested test was working.